pages/urls/views/elements.py

```diff
@@ -24,9 +24,12 @@
 
 from ... import settings
 from ...compat import re_path
-from ...views.elements import PageElementView
+from ...views.elements import PageElementView, CertificateDownloadView
 
 urlpatterns = [
+    re_path(r'^(?P<sequence_slug>%s)/certificate' % settings.SLUG_RE,
```

Comment:
Add a '/' to the URL. By convention, API endpoints: no trailing '/', Pages/Downloads: trailing '/'.